caom2pipe/reader_composable.py

```diff
@@ -114,6 +114,11 @@ def __init__(self):
         self._file_info = {}  # cadcdata.FileInfo
         self._logger = logging.getLogger(self.__class__.__name__)
 
+    def __str__(self):
+        file_info_keys = ''.join(ii for ii in self._file_info.keys())
```

Comment:
No white space between the keys here? Just checking.

Reply:
Added whitespace.